flutter-idea/src/io/flutter/run/ObservatoryFile.java

```diff
@@ -175,13 +175,10 @@ ObservatoryFile downloadOrGet(@NotNull String scriptId, boolean wantSnapshot) {
     }
   }
 
-  private static class Position {
-    final int line; // zero-based
-    final int column; // zero-based
-
-    Position(int line, int column) {
-      this.line = line;
-      this.column = column;
-    }
+  /**
+   * @param line   zero-based
+   * @param column zero-based
+   */
+  private record Position(int line, int column) {
```

Comment:
This is a good example of the win. Those fields are just boilerplate. (And the `final` keywords too.)